Localization/zh-Hans.lproj/Localizable.strings

```diff
@@ -13,7 +13,7 @@
 "5th" = "5级";
 "6th" = "6级";
 "7~9th" = "7～9级";
-"all" = "全部";
+"all" = "等级";
```

Comment:
The reason for this change. ↓
https://github.com/Kota1021/VocabDiaryLocalizable/pull/2/files#r1234555035